cookbooks/wazuh_elastic/attributes/elasticsearch.rb

```diff
@@ -17,7 +17,7 @@
 default['wazuh-elastic']['elasticsearch_path_data'] = "/var/lib/elasticsearch"
 default['wazuh-elastic']['elasticsearch_path_logs'] = "/var/log/elasticsearch"
 default['wazuh-elastic']['elasticsearch_port'] = 9200
-default['wazuh-elastic']['elasticsearch_ip'] = '<YOUR_ELASTICSEARCH_IP>'
+default['wazuh-elastic']['elasticsearch_ip'] = '127.0.0.1'
```

Comment:
I think we should keep the `<YOUR_ELASTICSEARCH_IP>` variable. Otherwise, if a user has not Elasticsearch configured for listening in the loopback interface, could be messy. Let's open a new issue for this.